ci-operator/config/openshift/tektoncd-cli/openshift-tektoncd-cli-release-next.yaml

```diff
@@ -0,0 +1,33 @@
+tag_specification:
+  name: '4.0'
```

Comment:
I think you should at least use 4.1 - we haven't even released anything 4.0 AFAIK